### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by LanaDX**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,23 +1,98 @@
...
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
 
-* [Default.aspx](./CS/Default.aspx) (VB: [Default.aspx](./VB/Default.aspx))
-* [Default.aspx.cs](./CS/Default.aspx.cs) (VB: [Default.aspx.vb](./VB/Default.aspx.vb))
-<!-- default file list end -->
-# ASPxGridView - How to set text wrapping in different ways
+# Grid Control for ASP.NET Web Forms - How to set text wrapping in different ways
```
Comment (arminal) [inline Readme.md]: How to set text wrapping in different ways --> this phrasing won't work, can you rewrite this? Maybe Different options for wrapping text or something else...

File: Readme.md [hunk 2]
```diff
@@ -1,23 +1,98 @@
...
 <!-- run online -->
 **[[Run Online]](https://codecentral.devexpress.com/t175881/)**
 <!-- run online end -->
 
+The suggested techniques are common and may be used not only for [ASPxGridView](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridView) but for other controls in the same way.
```
Comment (arminal) [inline Readme.md]:
```suggestion
The suggested techniques are common and may be used not only for [ASPxGridView](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridView), but for other controls also.
```

File: Readme.md [hunk 3]
```diff
@@ -1,23 +1,98 @@
...
+      
+    ```ASPx
+    <dx:GridViewDataColumn FieldName="C2" Width="100">  
+        <CellStyle CssClass="disableWrapping" />
+    </dx:GridViewDataColumn>
+    ```
+
+Note that when text wrapping is disabled, the column's `Width` property in not in effect.
+
+## Enable wrapping to have a fixed column width
```
Comment (arminal) [inline Readme.md]:
```suggestion
## Enable wrapping to set column width
```

File: Readme.md [hunk 4]
```diff
@@ -1,23 +1,98 @@
...
+    
+    ```ASPx
+    <dx:GridViewDataColumn FieldName="C4" Width="100">  
+        <CellStyle CssClass="enableWrapping" />  
+    </dx:GridViewDataColumn>  
+    ```
+
+## Wrap text without white spaces
+
+The previous approach works if a cell text contains white spaces only. To wrap text that does not contain white spaces, use the CSS `word-break` property.
```
Comment (arminal) [inline Readme.md]:
```suggestion
The previous approach works if cell text contains white spaces only. To wrap text that does not contain white spaces, use the CSS `word-break` property.
```

File: Readme.md [hunk 5]
```diff
@@ -1,23 +1,98 @@
...
+``` 
+```ASPx
+<dx:GridViewDataColumn FieldName="C5" Width="100">  
+    <CellStyle CssClass="wrapEmail" />
+</dx:GridViewDataColumn>  
+```
+
+## Show a truncated text
+
+Set the [ASPxGridView.SettingsBehavior.AllowEllipsisInText](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridBehaviorSettings.AllowEllipsisInText) property to to `true` to automatically truncate cell texts if they do not fit into the cell width. To indicate that the text is clipped, the control displays an ellipsis (`…`).
```
Comment (arminal) [inline Readme.md]:
```suggestion
Set the [ASPxGridView.SettingsBehavior.AllowEllipsisInText](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridBehaviorSettings.AllowEllipsisInText) property to `true` to automatically truncate cell text if it does not fit into the cell's width. To indicate clipped text, the control displays an ellipsis (`…`).
```

File: Readme.md (hunk 1 above)
Comment (LanaDX) [reply Readme.md]:
```suggestion
# Grid Control for ASP.NET Web Forms - Different options for wrapping text
```

**Pull request merged**